Dockerfile.mysql

```diff
@@ -0,0 +1,10 @@
+FROM mysql:9
+
+ENV MYSQL_DATABASE=app_db
+ENV MYSQL_USER=app_user
+ENV MYSQL_PASSWORD=1234
+ENV MYSQL_ROOT_PASSWORD=1234
```

Comment:
Using '1234' as a root password is insecure. Consider using a stronger password for MYSQL_ROOT_PASSWORD.